reference/runtimes.md

```diff
@@ -1,16 +1,16 @@
 # Runtimes
 
-Adobe I/O Runtime supports Node.js both versions 10 and 12. The following npm modules are pre-installed (if your action uses any of these modules, you don&rsquo;t have to package them together with action code):
+Adobe I/O Runtime supports Node.js versions 14, 12 and 10. The following npm modules are pre-installed (if your action uses any of these modules, you don&rsquo;t have to package them together with action code):
 
-### Node.js v10
+### Node.js v14
 
-    "express": "4.16.4",
-    "openwhisk": "3.21.2",
-    "body-parser": "1.18.3",
+    "express": "4.17.1",
+    "openwhisk": "3.21.3",
+    "body-parser": "1.19.0",
     "cls-hooked": "4.2.2",
-    "redis": "2.8.0",
-    "request": "2.88.0",
-    "request-promise": "4.2.2"
+    "redis": "3.1.2",
+    "node-fetch": "2.6.1"
+    "dnscache": "1.0.2"
 
```

Comment:
missing dnscache dependency and couple typos
PR for fix: https://github.com/AdobeDocs/adobeio-runtime/pull/79